quickstarts/synthetics/free_tier/config.yml

```diff
@@ -0,0 +1,32 @@
+id: 260027d0-9d8f-4a35-a676-73b5ee6c0d2d
+# Name of the quickstart (required)
+slug: free-tier
+
+# Displayed in the UI (required)
+title: Free Teir
```

Comment:
It looks like Tier is not spelled correctly. Also, we might want a little more descriptive about what this free tier is _for_. Perhaps something like `Mona Labs`?